architecture-building-systems/CityEnergyAnalyst
# Update jenkins documentation

This PR updates the documentation on how to renew python dependencies on the Jenkins server (aka, the PC sitting in HIB).

To test:
- read the modified parts of the documentation and make sure everything is clear.
- make sure you are able to locate the folder `C:\ProgramData\ceajenkins` on the Jenkins server.
